spec/defines/jenkins_job_spec.rb

```diff
@@ -1,5 +1,8 @@
 require 'spec_helper'
 
+Puppet::Util::Log.level = :debug
+Puppet::Util::Log.newdestination(:console)
```

Comment:
It's not needed at all. It's debugging code that didn't get removed before I pushed....